tests/unit/customizations/s3/test_utils.py

```diff
@@ -653,6 +653,28 @@ def test_upload_part_copy(self):
              'SSECustomerKey': 'my-sse-c-key'})
 
 
+class TestRequestParamsMapperChecksumAlgorithm:
+    @pytest.fixture
+    def cli_params(self):
+        return {'checksum_algorithm': 'CRC32'}
+
+    def test_put_object(self, cli_params):
+        request_params = {}
+        RequestParamsMapper.map_put_object_params(request_params, cli_params)
+        assert request_params == {'ChecksumAlgorithm': 'CRC32'}
+
+
+class TestRequestParamsMapperChecksumMode:
```

Comment:
Let's add a test case for empty `cli_params` and assert `ChecksumAlgorithm` isn't present.

Reply:
Good point thank you, resolving in next revision